test/src/test/java/org/corfudb/integration/CorfuReplicationSiteConfigIT.java

```diff
@@ -18,6 +18,7 @@
 public class CorfuReplicationSiteConfigIT extends AbstractIT {
     final static int MAX_RETRY = 10;
     final static long sleepInterval = 10000;
+    private final String pluginConfigFilePath = "src/test/resources/transport/nettyConfig.properties";
```

Comment:
Nan would you mind adding a brief description on what this test does. So when someone wants to look at it they know in plain English what it is doing. Maybe something in the line: Bring up an active and standby Log Replication Server, start replication. Initiate a cluster role switch in the middle of (snapshot/log entry) sync, while data is still being persisted into the active datastore (or do you stop the writing before the switch).. is the replication still going on when asked to switch or did it finish already? something in that line so we can easily detect which tests we need to add.

Reply:
Yeah, we definitely need a description. I will add descriptions for new tests.

Now any IT run replication as a thread will fail in Travis CI, and the current default cluster manager is not able to change the topology config unless running as a thread. 

So I am introducing a new test cluster manager that subscribes a stream and updates config onNext(). In this way, we can easily control config change in test, and new cluster config IT can run as a process and pass Travis build.

I keep this one to verify config change logic, and will remove it after my new cluster config complete.

Reply:
ok, yes I was thinking on this test and I have some thoughts on how to run it in processes. I'll share it with you offline I think it will let us model several scenarios very quickly.